### PR TITLE
Translated README.md into French resolving #190 in addition to #391

### LINKED ISSUE #190: Add translations for Readme.md
**Actual Behaviour**

The readme.md only appears in English. Some of us not familiar with English.

**Expected Behaviour**

Add translations of readme.md in other languages and add links into original readme.md

**Steps to reproduce it**

Make another folder to locate these files and add your translations of readme.md into it.

**Would you like to work on the issue?**

All of us, need to get together for work.

### PR DESCRIPTION
**Pull request by Ritzing**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

Preview Link : https://github.com/Ritzing/gci18.fossasia.org/blob/master/translations/FrenchReadme.md

- Fixes #190 

## Description / Changes
Translated README.md into French resolving #190  in continuation with #391

### After:
ss1
ss2
ss3

- - - - - - - - - - - -

### CODE REVIEW

**Comment by Ritzing** [comment]
Added Screenshot and preview link

**Comment by jakubpawlina** [comment]
Add yet a comma between German and French or something like this (eg. order in points) in original README.md file

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ FOSSASIA Google Code-In Website 2018
 
 This is the repository for FOSSASIA's website for Google Code-In 2018, we at FOSSASIA intend to develop it collaboratively during the course of this competition by participating students themselves. Fork the repository before making changes and make sure you read [FOSSASIA Best Practices](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/)
 
-Read in **[German](https://github.com/Ritzing/gci18.fossasia.org/blob/master/translations/GermanReadme.md)**
+Read in **[German](https://github.com/Ritzing/gci18.fossasia.org/blob/master/translations/GermanReadme.md)** ,
```
Comment (jakubpawlina) [inline README.md]: Remove the space before comma

**Comment by Ritzing** [comment]
@sk9331657 @jakubpawlina @liveHarshit  all changes made as requested , thanks for the review

**Comment by liveHarshit** [comment]
Commits are not squashed.

**Pull request merged**